web/frontend/src/views/guild/tabs/AutoKickerTab.vue

```
@@ -0,0 +1,114 @@
+  error.value = null;
+  config.value = null;
+  try {
+    const next = await api.get<AutoKickerConfig>(`/guilds/${props.guildId}/auto-kicker`);
+    if (seq !== _loadSeq) return;
+    if (next.kick_after < 1) next.kick_after = 1;
+    config.value = next;
+  } catch (e: unknown) {
+    if (seq !== _loadSeq) return;
+    error.value = e instanceof Error ? e.message : "Failed to load";
+  } finally {
+    if (seq !== _loadSeq) return;
```

Biome (2.4.6)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)